doc/dftb+/manual/dftbp.tex

```diff
@@ -5524,15 +5523,18 @@ \subsection{Casida}
     created, which contains the spectrum at the uncoupled DFTB level (i.e.\ the
     single-particle excitations).
 
-  \item[\is{WriteTransitionCharges}] For range separated calculations
-    (p.~\pref{sec:dftbp.Hybrid}) setting this to \is{Yes} generates the file
-    ATQ.DAT containing the Mulliken charges for a specific transition (chosen by
-    \is{StateOfInterest}).
+  \item[\is{WriteTransitionCharges}] For singlet or spin-polarized
+    calculations setting this to \is{Yes} generates the file
+    ATQ.DAT containing the Mulliken charges for a specific excited state (chosen by
```

Comment:
Is "transition" less ambiguous than "excited state", as this isn't an actual excited state set of charges (or did I misunderstand)?

Reply:
Transition is incorrect, as charges are for a many-body state and not transition charges.